deps/ccommon/include/buffer/cc_buf.h

```diff
@@ -68,7 +68,7 @@ struct buf {
     char              *wpos;    /* write marker */
     char              *end;     /* end of buffer */
     bool              free;     /* is this buf free? */
-    char              begin[1]; /* beginning of buffer */
+    char              begin[];  /* beginning of buffer */
```

Comment:
@thinkingfish will this cause any issues?

Reply:
One note here: using `begin[1]` is technically undefined behaviour. Idk if any compilers exploit it but using `begin[]` is explicitly supported by the C standard.

A second note: I didn't find any code using `sizeof(buf)` anywhere within either pelikan or ccommon.

Reply:
I'm almost sure the incorrect declaration is a copy of how `struct item` is defined in the slab module in twemcache (https://github.com/twitter/twemcache/blob/master/src/mc_items.h#L134), which came from memcache

Reply:
Okay with the change here then